core/local/chokidar/event_buffer.js

```diff
@@ -76,7 +76,7 @@ class EventBuffer /*:: <EventType> */ {
     if (this.events.length > 0) {
       const flushedEvents = this.events
       this.events = []
-      this.flushed(flushedEvents)
+      return this.flushed(flushedEvents)
```

Comment:
Why the `return` here? I'm surprised because it doesn't match the signature: `flush() /*: void */`

Reply:
I need to be able to wait for the flush to complete in the scenarios runner but the result of `this.flushed()` which is an async function was not returned by `flush()`.
So I decided to return it. It has no effect on the production code but I forgot to update the method's signature.